rero_ils/modules/cli.py

```diff
@@ -35,15 +35,30 @@
 from invenio_records_rest.utils import obj_or_import_string
 from invenio_search.cli import es_version_check
 from invenio_search.proxies import current_search
+from sqlalchemy import MetaData, create_engine
 from werkzeug.local import LocalProxy
 
+from .holdings.api import Holding
 from .items.cli import create_items, reindex_items
 from .loans.cli import create_loans
 from .patrons.cli import import_users
 
 _datastore = LocalProxy(lambda: current_app.extensions['security'].datastore)
 
 
+def append_pid_to_table(table, pids):
```

Comment:
Maybe use `append_pids_to_table` for this function.
Very similar code to `rero_ils/modules/providers.py :append_pid_to_table`

Reply:
Will put a new pull request to address this and testing coverage later.